src/thunderbots/software/ai/world/game_state.h

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <geom/point.h>
+#include <util/refbox_constants.h>
+
+/**
+ * @brief Holds the state of the game according to the referee
+ *
+ * @details Contains information on what period of the game it is, what type of
+ * play to run, team scores, time remaining, etc.  During normal gameplay, the
+ * information in this class is received from the [ssl
+ * refbox](https://github.com/RoboCup-SSL/ssl-refbox)
+ * program over the network.
+ *
+ * Taken from
+ */
+class GameState
+{
+   public:
+    enum State
+    {
+        HALT,    // Robots must not move
+        STOP,    // Robots must stay 500mm away from ball
+        SETUP,   // Robots not on starting team must stay 500mm away from ball
+        READY,   // A robot on the starting team may kick the ball
+        PLAYING  // Normal play
+    };
+
+    // Types of restarts
+    enum Restart
```

Comment:
Could we change this to:
```suggestion
    enum RestartReason
```
?
I think that it would be a bit more clear, as someone not intimately familiar with the robocup rules.

Reply:
agree

Reply:
bump

Reply:
bumping this again. `Restart` -> `RestartReason` is a bit more clear what this enum is.

Reply:
fixed